src/scripts/ic7/shared/htm_report.py

```diff
@@ -453,6 +453,40 @@ def get_combined_stats(self) -> dict[str, Any]:
             "Total Deaths averted (covid disruption) for hiv 2021 to 2026": total_deaths_averted,
         }
 
+    def comb_mort(self) -> pd.DataFrame:
+        """Generate graphs for the combined mortality. """
+
+        df = self._calculate_combined_mortality_stats()
+
+        return df['Combined mortality df']
+
+    def comb_inc(self) -> pd.DataFrame:
+        """Generate graphs for the combined incidence. """
+
+        df = self._calculate_combined_incidence_stats()
+
+        return df['Combined incidence df']
```

Comment:
the use of suffix 'df' is possibly a bit confusing as what we get back from `_calculate_combined_incidence_stats()` is a Dict.
We could simply this to just
```suggestion
    return self._calculate_combined_incidence_stats()['Combined incidence df']
```